frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -309,6 +317,7 @@ class Dashboard extends Component {
                     topNav={embedOptions?.top_nav}
                   >
                     {parametersWidget}
+                    <FilterApplyButton />
```

Comment:
Use a dedicated component that connects to all the necessary data so we don't need to pass any prop to it.